README.md

```diff
@@ -139,6 +124,12 @@ Example command:
 
 Clone this repository to your desired folder:
 
+1.click: code and copy the link
+
+2.run: git clone git @ https://github.com/AyomipoMajek/To-Do-List
+
+2.Open the file in your code editor
+
```

Comment:
- [ ] Please make sure you add all the correct and complete steps up to the point where the app is running correctly.